src/Domain/Creators/Shell.hpp

```diff
@@ -49,6 +49,32 @@ class Shell : public DomainCreator<3> {
       CoordinateMaps::ProductOf2Maps<CoordinateMaps::Affine,
                                      CoordinateMaps::Identity<2>>>>;
 
+  /// Options for the EquatorialCompression map
+  struct EquatorialCompressionOptions {
+    static constexpr Options::String help = {
+        "Options for the EquatorialCompression map, used for modifying the "
+        "angular distribution of gridpoints."};
+    struct AspectRatio {
+      using type = double;
+      static constexpr Options::String help = {
+          "The equatorial compression factor."};
+    };
+    struct IndexPolarAxis {
+      using type = size_t;
+      static constexpr Options::String help = {
+          "The index (0, 1, or 2) of the axis along which equatorial "
+          "compression "
+          "is applied."};
+    };
+    using options = tmpl::list<AspectRatio, IndexPolarAxis>;
+    EquatorialCompressionOptions() = default;
+    EquatorialCompressionOptions(double aspect_ratio, size_t index_polar_axis)
+        : aspect_ratio_(aspect_ratio), index_polar_axis_(index_polar_axis) {}
```

Comment:
I think these can be deleted.

Reply:
Do you mean the entire
```
EquatorialCompressionOptions() = default;
EquatorialCompressionOptions(double aspect_ratio, size_t index_polar_axis)
        : aspect_ratio_(aspect_ratio), index_polar_axis_(index_polar_axis) {}
```
block? Would `aspect_ratio_` and `index_polar_axis_` still be initialized correctly with this change?